verilog/formatting/formatter_test.cc

```diff
@@ -6963,6 +6963,30 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      "    type TT = my_pkg::my_type2_t\n"
      ");\n"
      "endinterface\n"},
+     {// wildcard import package at module header
+     "module foo import bar::*; (baz); endmodule\n",
+     "module foo\n"
+     "  import bar::*;\n"
+     "(\n"
+     "    baz\n"
+     ");\n"
+     "endmodule\n"},
+     {// import package at module header
+     "module foo import bar::baz; (qux); endmodule\n",
+     "module foo\n"
+     "  import bar::baz;\n"
+     "(\n"
+     "    qux\n"
+     ");\n"
+     "endmodule\n"},
+     {// wildcard import mutiple packages at module header
+     "module foo import bar::*,baz::*; (qux); endmodule\n",
+     "module foo\n"
+     "  import bar::*, baz::*;\n"
+     "(\n"
+     "    qux\n"
+     ");\n"
+     "endmodule\n"},
```

Comment:
request for one more test case with two separate imports, like:

```systemverilog
module foo
  import bar1::*;
  import bar2::quux;
...
```

Reply:
This will cause a syntax error. Should I add this to the parser?

Reply:
Oh, my mistake.  In that case, this PR is done.  I'll approve we'll take it from here.

Reply:
@fangism Should I open a new issue for the separate imports?

Reply:
I tested your change, and found that it already works for me, so I'll add tests to cover this myself.  Thanks though!

Reply:
If there is anything else that I should do, please let me know!

Reply:
This one is done, just waiting for colleague to merge.